lib/memfs/io.rb

```diff
@@ -18,6 +18,20 @@ def read(path, *args)
       ensure
         file.close if file
       end
+
+      def write(path, string, offset = 0, open_args = nil)
+        open_args ||= [File::WRONLY, encoding: nil]
+
+        if offset > 0
+          fail NotImplementedError, "MemFs::IO.write with offset not yet supported."
+        end
+
+        file = open(path, *open_args)
+        file.seek(offset || 0)
```

Comment:
I'm not completely comfortable with this line since passing `false` should result in `no implicit conversion from boolean (TypeError)` an here it would end up being zero.